pkg/cloud/aws/services/wait/wait.go

```diff
@@ -19,7 +19,8 @@ package wait
 import (
 	"time"
 
-	aMW "k8s.io/apimachinery/pkg/util/wait"
+	"github.com/pkg/errors"
+	apiMachineryWait "k8s.io/apimachinery/pkg/util/wait"
```

Comment:
Import aliases should be all lowercase. Alternatively, we don't need an alias here. A reference to `wait.Backoff` below means the apimachinery package, not the current package.

If we want to keep an alias, I usually do something like `kwait` for kubernetes wait.